sys/include/fs/littlefs2_fs.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_littlefs2 littlefs v2 integration
+ * @ingroup     pkg_littlefs2
+ * @brief       RIOT integration of littlefs version 2.x.y
+ *
+ * @{
+ *
+ * @file
+ * @brief       littlefs v2 integration with vfs
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef FS_LITTLEFS2_FS_H
+#define FS_LITTLEFS2_FS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "vfs.h"
+#include "lfs.h"
+#include "mtd.h"
+#include "mutex.h"
+
+/**
+ * @name    littlefs configuration
```

Comment:
```suggestion
 * @name    littlefs configuration
 * @ingroup config
```
Also, it would be good to  `CONFIG_`-prefix them now, rather that change it later when exposed to Kconfig

Reply:
Good one, I've added the group and I prefixed all configuration settings with `CONFIG_`.